specification/dimensions/quantityinusageunit.md

```diff
@@ -0,0 +1,32 @@
+# Quantity In Usage Unit
```

Comment:
As we've seen columns mature and close on a common prefixed grouping pattern, I would really like to see this and `QuantityInPricingUnit` be called `UsageQuantity` and `PricingQuantity` so they sit next to their respective unit columns in a list. These will be very out-of-place if not sitting next to the column that explains what the number means.